kencu@macports.org

Needed to run the test suite with the default toolchain on older darwin systems.
Removes flags unsupported by gcc-4.2

https://trac.macports.org/ticket/61170

--- ./testsuite/lib/libffi.exp.orig
+++ ./testsuite/lib/libffi.exp
@@ -488,7 +488,7 @@
         }
       }
       "gnu" {
-        set common "-W -Wall -Wno-psabi"
+        set common "-W -Wall"
         if [info exists env(LIBFFI_TEST_OPTIMIZATION)] {
 	  set optimizations [ list $env(LIBFFI_TEST_OPTIMIZATION) ]
         } else { 
--- ./testsuite/libffi.bhaible/bhaible.exp.orig
+++ ./testsuite/libffi.bhaible/bhaible.exp
@@ -24,7 +24,7 @@
 # was done in a pretty lazy fashion, and requires the use of compiler
 # flags to disable warnings for now.
 if { [string match $compiler_vendor "gnu"] } {
-    set warning_options "-Wno-unused-variable -Wno-unused-parameter -Wno-unused-but-set-variable -Wno-uninitialized";
+    set warning_options "-Wno-unused-variable -Wno-unused-parameter -Wno-uninitialized";
 }
 if { [string match $compiler_vendor "microsoft"] } {
     # -wd4996  suggest use of vsprintf_s instead of vsprintf
